### ISSUE TITLE
Upgrade alpine images to 3.14

### ISSUE THREAD

**Issue by wanghuizzz** [openresty/docker-openresty#187]
@neomantra 
Alpine 3.14 has been released, could you upgrade alpine images to 3.14 please?

**Comment by neomantra** [comment]
You can try your own build by cloning this repo and running:

```
docker build -f alpine/Dockerfile --build-arg RESTY_IMAGE_TAG=3.14 .
```

I just built it successfully, but have no testing.   As there are not upstream OpenResty packages available yet (only build-from-source like that) and they do a pretty good job of coverage testing, I'm going to wait a while before releasing something new.

I will also note that Alpine 3.14 switched its LuaJIT to [OpenResty's branch](https://gitlab.alpinelinux.org/alpine/aports/-/commit/c12fb28e6d794fa1cb9ceda035d06edbff432c29).

**Comment by neomantra** [comment]
After waking up a bit, I reconsidered this... the policy before has been to advance the build-from-source `alpine` with upstream Alpine and then `alpine-apk` with upstream OpenResty.   In general, the build-from-source is more experimental or custom.  So I just pushed https://github.com/openresty/docker-openresty/commit/4bb66816d75b2512999e3ea019aa498cfcdc3425 and will tag when CI is successful.

**Comment by neomantra** [comment]
Tagged as `1.19.3.2-3`

**Comment by neomantra** [comment]
While working on #188, I realized that the `alpine` Dockerfile was incorrectly updated and `1.19.3.2-3` has Alpine 3.13. 

I'm running into an issue that Travis cannot build Alpine 3.14, I think due to these;
 * https://wiki.alpinelinux.org/wiki/Release_Notes_for_Alpine_3.14.0#faccessat2
 * https://gitlab.alpinelinux.org/alpine/aports/-/issues/12396

Once I get this fixed, I'll release a `1.19.3.2-4`.

**Comment by neomantra** [comment]
This never got done and we've moved on to Alpine 3.15  with ` 1.19.9.1-4`....
